### DISCUSSION TITLE
patchClientRoutes的路由如何像静态路由一样 做路由懒加载 有推荐的方式嘛

### DISCUSSION THREAD

**Discussion by ErnestWangZuNian** [umijs/umi#11627] [Q&A]
Umi 4 默认按页拆包，从而有更快的页面加载速度 静态路由确实是这样 如果是patchClientRoutes动态添加的路由如何实现路由懒加载 自己去懒加载包 官方有推荐的方式嘛

**Comment by fz6m** [comment]
See #11626

**Reply by penny006** [reply]
请问这个方法适用umi3不?路由配element而不是component我试过了，打包启动都不报错，就是页面打开白屏

**Reply by fz6m** [reply]
umi 3 不维护了，建议升级到 umi 4
